### PR TITLE
Use CommonHTML output and automate mathjax cleanup

### LINKED ISSUE #30: Upgrade MathJax to 2.7.2 and update config
As much as possible, our custom build of mathjax electron should match the `TeX-MML-AM_CHTML` config with MathJax. That's certainly not the case right now as I found out in https://github.com/nteract/nteract/pull/2350

/cc @mpacer

### PR DESCRIPTION
**Pull request by lgeiger**
Closes #30 

This also exports `mathJaxPath` to make it easier to integrate with `@nteract/mathjax`.

bildschirmfoto 2018-10-11 um 14 35 19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-[![Build Status](https://travis-ci.org/nteract/mathjax-electron.svg?branch=master)](https://travis-ci.org/nteract/mathjax-electron) [![Greenkeeper badge](https://badges.greenkeeper.io/nteract/mathjax-electron.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/nteract/mathjax-electron.svg?branch=master)](https://travis-ci.org/nteract/mathjax-electron)
 
 A trimmed down version of the [MathJax](https://www.mathjax.org/) library for use with electron and modern browsers.
 
 `mathjax-electron` allows you to render math inside your application while keeping the package size at a minimum. To achieve this we provide a preconfigured MathJax environment with only the necessary bits of the MathJax Library included.
 
-The package size is 2.9 MB compared to 66 MB for a full MathJax installation. If you need support for different output formats and legacy browsers try [`mathjax-node`](https://github.com/mathjax/MathJax-node).
+The package size is ~1 MB compared to 66 MB for a full MathJax installation. If you need support for different output formats and legacy browsers try [`mathjax-node`](https://github.com/mathjax/MathJax-node).
```
Comment (lgeiger) [inline README.md]: Now only 1 MB 🎉

Comment (rgbkrk) [reply README.md]: Went even _smaller_? How is that possible?!

We're not missing some fonts are we?

**Comment by rgbkrk** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (lgeiger) [reply README.md]: I think we included unnecessary fonts before. I haven't encountered any problems yet.

Comment (rgbkrk) [reply README.md]: Awesome. At some point in the future we should probably make a suite of integration tests that we can evaluate at a glance how things are rendering (if we're supporting what people expect).

**Pull request merged**